src/components/Form/ServiceForm.tsx

```diff
@@ -101,32 +97,41 @@ function ServiceForm() {
           token.decimals,
         );
         const parsedRateAmountString = parsedRateAmount.toString();
-        const cid = await postToIPFS(
-          JSON.stringify({
+
+        let tx, cid;
+
+        cid = await tlClient?.service?.updloadServiceDataToIpfs(
```

Comment:
why this one got a different name updloadServiceDataToIpfs vs just upload for the others ?